docs/Protected_build_porting_guide.md

```diff
@@ -0,0 +1,363 @@
+# Protected Build Porting Guide
```

Comment:
Protected build is a build feature. Porting guide can't be for a build feature.

Reply:
@Nishtha97 
Let us rename this document as "HowToSupportTizenRtProtectedBuild.md".
And let us provide a link to this document from Porting_Guide.md

Reply:
Document is renamed